wlog/auditlog/audit2log/context.go

```diff
@@ -32,6 +37,22 @@ func WithLogger(ctx context.Context, logger Logger) context.Context {
 	return context.WithValue(ctx, contextKey, logger)
 }
 
+// WithLoggerParams returns a copy of the provided context whose logger is configured with the provided parameters. If
+// no parameters are provided, the original context is returned unmodified. If the provided context did not have a
+// logger set on it, the returned context will contain the default logger configured with the provided parameters. If
+// any of the provided parameters set safe or unsafe values, the returned context will also have those values set on it
+// using the wparams.ContextWithSafeAndUnsafeParams function.
+func WithLoggerParams(ctx context.Context, params ...Param) context.Context {
+	if len(params) == 0 {
+		return ctx
+	}
+	// if the provided params set any safe or unsafe values, set those as wparams on the context
+	if safeParams, unsafeParams := safeAndUnsafeParamsFromParams(params); len(safeParams) > 0 || len(unsafeParams) > 0 {
```

Comment:
audit logs don't have safe/unsafe (as mentioned below) so this block is not necessary